lib/phoenix/code_reloader/server.ex

```diff
@@ -178,14 +192,31 @@ defmodule Phoenix.CodeReloader.Server do
     Mix.Project.config[:consolidate_protocols]
   end
 
-  defp proxy_io(fun) do
+  defp proxy_io(fun, callback) do
     original_gl = Process.group_leader
-    {:ok, proxy_gl} = Proxy.start()
+    {:ok, proxy_gl} = Proxy.start(callback)
     Process.group_leader(self(), proxy_gl)
+    stderr =
+      case Process.whereis(:standard_error) do
+        nil ->
+          nil
+        pid ->
+          Process.unregister(:standard_error)
```

Comment:
This is not safe. Imagine you have processes running alongside your Phoenix application and they decide to write something to stderr, they can now randomly fail.